DigitalLearningSolutions.Web/Controllers/Support/RequestSupportTicketController.cs

```diff
@@ -234,23 +234,30 @@ public IActionResult DeleteImage(DlsSubApplication dlsSubApplication, string ima
         [Route("/{dlsSubApplication}/RequestSupport/SupportSummary")]
         public IActionResult SupportSummary(DlsSubApplication dlsSubApplication, SupportSummaryViewModel supportSummaryViewModel)
         {
+            if (!TempData.Any())
+            {
+                return RedirectToAction("StatusCode", "LearningSolutions", new { code = 401 });
+            }
             var data = multiPageFormService.GetMultiPageFormData<RequestSupportTicketData>(
                 MultiPageFormDataFeature.AddCustomWebForm("RequestSupportTicketCWF"),
                 TempData
-            ).GetAwaiter().GetResult(); ;
+            ).GetAwaiter().GetResult(); 
             var model = new SupportSummaryViewModel(data);
             return View("SupportTicketSummaryPage", model);
         }
 
         [HttpPost]
         [Route("/{dlsSubApplication}/RequestSupport/SubmitSupportSummary")]
         public IActionResult SubmitSupportSummary(DlsSubApplication dlsSubApplication, SupportSummaryViewModel model)
-
         {
+            if (!TempData.Any())
+            {
+                return RedirectToAction("StatusCode", "LearningSolutions", new { code = 401 });
```

Comment:
A 401 error on the front end does not look good for the user when it says "something is wrong" - it suggests that the page is broken. We should show a 410 error instead. Look at the journey where we send a confirmation request on a self-assessment and after the submit button, if we press the back button to get inspiration.

Reply:
Done